### DISCUSSION TITLE
Microsoft Update Release Notification

### DISCUSSION THREAD

**Discussion by TravisEz13** [PowerShell/PowerShell#16459] [General]
Please give feedback, but we think we will keep this discussion up to date with Microsoft Update release dates.  At first this will probably be as they happen, until we are confident, we can hit our planned dates.

**Comment by thlac** [comment]
In case anybody is watching this issue, it appears that "Powershell - x64" is now available as a product in WSUS, with 7.2.1 available 👍

**Comment by TravisEz13** [comment]
We are targeting the Microsoft Update for 7.2.2 to go live on March 28th.  --We are still working out some issues in the publishing process so please be patient if there are issues.--

Update - 2022-03-31 : The update went live on 3/28 as planned, less than 2 weeks after 7.2.2 was released to GitHub.  This is within the time we expect updates to occur.

**Reply by doctordns** [reply]
I would like 2 things here. First a faster publishing time via MU. A month is a very long time. If msft can publish Defender updates 'immediately', we can do better.  Second,  sync the in product message with MU availability. I have MU setup, so when I see the message, MU should be on the case.
